_includes/k8s/bestpractices.md

```diff
@@ -0,0 +1,10 @@
+
+## StorageOS Pod placement
+
+## Resource reservations
+
+## Network configuration
+
```

Comment:
Storage Management
Use of External Etcd

Reply:
I had this in mind. ETCD added already and adding storage now

Reply:
done